locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed by
+[nvm](https://github.com/creationix/nvm) by installing into the location that nvm expects:
+
+```bash
+$ make install DESTDIR='$(NVM_DIR)/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
The function itself is POSIX-compliant, so it should work in any POSIX-compliant shell.

I think that it depends on if your goal is correctness/robustness or convenience. Correctness requires using the same `NVM_DIR` env var that the user will use, and robustness requires not hardcoding the nvm dir's structure. Could you run a separate bash script that sources nvm.sh, and output the version path, and run that from the Makefile?